mock/context/outputContextStub.go

```diff
@@ -159,9 +159,9 @@ func (o *OutputContextStub) Transfer(destination []byte, sender []byte, gasLimit
 }
 
 // TransferESDT mocked method
-func (o *OutputContextStub) TransferESDT(destination []byte, sender []byte, transfers []*vmcommon.ESDTTransfer, callInput *vmcommon.ContractCallInput) (uint64, error) {
+func (o *OutputContextStub) TransferESDT(destination []byte, originalCaller []byte, sender []byte, transfers []*vmcommon.ESDTTransfer, callInput *vmcommon.ContractCallInput) (uint64, error) {
```

Comment:
let's define some structure for this input - like ArgsTransfer or something like that. Too many arguments in input - it is a general refactor - can be done in the next PR.

Reply:
fixed